examples/kafka/cifar10/cifar10_kafka.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>It seems confluent_kafka is a dependency, may be worth adding a requirements.txt - also seems like to need tensorflow-serving-api</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>yes</p>